tools/Dockerfile

```diff
@@ -51,7 +51,9 @@ RUN apt-get update && apt-get install -y \
 	libgmp-dev \
 	libmpfr-dev \
 	libsuitesparse-dev \
-	libmpc-dev
+	libmpc-dev \
+	git
```

Comment:
Probably for another PR, but it can be helpful to sort these multiline apt-get docker installs alphabetically--this is suggested in the official [best practices guide](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments) to reduce the likelihood of accidental duplication/ease of updating in the future, etc.

Reply:
> Probably for another PR, but it can be helpful to sort these multiline apt-get docker installs alphabetically--this is suggested in the official [best practices guide](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments) to reduce the likelihood of accidental duplication/ease of updating in the future, etc.

I was thinking the same thing yesterday once I saw the lines on the github page-I'll reorder in this PR, it's a fairly small change to the PR. Thanks for calling this out @tylerjereddy